scripts/bundle-size-collect.js

```diff
@@ -0,0 +1,37 @@
+// This script collates bundle size information from
+// minified files in apps/test-bundles/dist and writes to
+// apps/test-bundles/dist/bundlesizes.json.
+// It is uploaded as an artifact by the build definition in
+// Azure Dev Ops and used to compare baseline and PR file size
+// information which gets reported by Size Auditor
+
+var fs = require('fs');
+
+var path = 'dist';
+var sizes = {};
+var outputFilePath = 'dist/bundlesizes.json';
+
+fs.readdir(path, function(err, items) {
```

Comment:
@kenotron since the other calls within this callback are synchronous, should we just prefer [`fs.readdirSync`](https://nodejs.org/api/fs.html#fs_fs_readdirsync_path_options) here?

Ultimately I don't think it'll impact the script much

Reply:
I usually prefer sync unless node is used as a server (express, etc).

Reply:
Agreed.